data.sql

```diff
@@ -1,3 +1,7 @@
 INSERT INTO animals (name) VALUES ('Agumon'), ('Gabumon'), ('Pikachu'), ('Devimon'), ('Charmander'), ('Plantmon'), ('Squirtle'), ('Angemon'), ('Boarmon'), ('Blossom');
 
-INSERT INTO vets (name) VALUES ('William Tatcher'), ('Maisy Smith'), ('Stephanie Mendez'), ('Jack Harkness');
\ No newline at end of file
+INSERT INTO vets (name) VALUES ('William Tatcher'), ('Maisy Smith'), ('Stephanie Mendez'), ('Jack Harkness');
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1800-01-01'::timestamp, '2021-01-01', '3 hours') visit_timestamp;
+
+INSERT INTO owners (full_name, email) select 'Owner ' || generate_series(1,9500000), 'owner_' || generate_series(1,9500000) || '@mail.com';
```

Comment:
- [ ] Kindly move `INSERT INTO` queries in the `schema.sql` because if your queries are in the `data.sql` and in the future someone tries to create a database and runs all queries from `schema.sql` they should have an empty database, only if someones really want to populate their database, they can use the `data.sql` file

Reply:
Hello, @DeliceLydia. Thanks for this observation. There seem not to be any `INSERT INTO` queries in the `schema.sql`. If you look closely, these lines that you've highlighted are from `data.sql`. The name of the file is at the top of this first review comment.

Reply:
oooh @Lameck1 sorry for this I confused it your queries are in the right place
thanks for reaching out!

I apologize for the inconvenience!